guides/rails_guides/markdown/renderer.rb

```diff
@@ -78,6 +81,7 @@ def lexer_language(code_type)
         end
 
         def clipboard_content(code, language)
+          # Remove prompt and results of commands.
```

Comment:
At a glance, it seems this this only removes the prompt, not the results.

```suggestion
          # Remove prompt of commands.
```

Reply:
Those lines get removed in the `gsub` on line 94.

Reply:
Ah, the grep eliminates the other lines (i.e. the results) as well, gotcha :+1: